js/ui/grid_core/ui.grid_core.editing.js

```diff
@@ -1167,19 +1240,24 @@ const EditingController = modules.ViewController.inherit((function() {
         },
 
         editCell: function(rowIndex, columnIndex) {
+            return this._editCell({ rowIndex, columnIndex });
+        },
+
+        _editCell: function(options) {
             const d = new Deferred();
             let coreResult;
 
             this.executeOperation(d, () => {
-                coreResult = this._editCellCore(rowIndex, columnIndex);
+                coreResult = this._editCellCore(options);
                 when(coreResult)
                     .done(d.resolve)
                     .fail(d.reject);
             });
 
             return coreResult !== undefined ? coreResult : d.promise();
         },
-        _editCellCore: function(rowIndex, columnIndex) {
+
+        _editCellCore: function({ rowIndex, columnIndex, oldRowIndex, oldColumnIndex } = {}) {
```

Comment:
I think that `= {}`  is not necessary here. The  _editCellCore method call without parameters wrong and impossible.

Reply:
Ok